lighthouse-core/gather/driver.js

```diff
@@ -628,8 +628,9 @@ class Driver {
     /**
      * @param {Driver} driver
      * @param {() => void} resolve
+     * @param {(error:Error) => void} reject
      */
-    function checkForQuiet(driver, resolve) {
+    function checkForQuiet(driver, resolve, reject) {
```

Comment:
WDYT about just catching the function call instead of adding another parameter here?

Reply:
I like it!  Couldn't get it to work before b/c I was stuck up on the `return;` never thought to _resolve_ them oops!